tests/lib/tools/tests.invariant

```diff
@@ -207,6 +208,17 @@ check_fakestore_cleaned() {
     fi
 }
 
+check_deleted_loop_devices() {
+	n="$1" # invariant name
+
+	losetup -l | grep -F ' (deleted)' > "$TESTSTMP/tests.invariant.$n"
```

Comment:
`(deleted)` is unlikely to shwo up in device name or file name, so we can be less picky about the format I think
```suggestion
	losetup -l | grep -F '(deleted)' > "$TESTSTMP/tests.invariant.$n"
```

Reply:
I think this part is relevant. The space is not quoted otherwise. I'd keep this.

Reply:
I did an experiment and I ran into another fragile part of the userspace API where escaping is just not done. The files are literally called like that, neither is deleted:

```
zyga@falka:~/snapd$ losetup -l
NAME       SIZELIMIT OFFSET AUTOCLEAR RO BACK-FILE                         DIO LOG-SEC
/dev/loop1         0      0         0  0 /home/zyga/snapd/potato (deleted)   0     512
/dev/loop0         0      0         0  0 /home/zyga/snapd/potato             0     512
zyga@falka:~/snapd$ cat /sys/class/block/loop[01]/loop/backing_file
/home/zyga/snapd/potato
/home/zyga/snapd/potato (deleted)
```

Reply:
```
zyga@falka:~/snapd$ losetup -l
NAME       SIZELIMIT OFFSET AUTOCLEAR RO BACK-FILE                                   DIO LOG-SEC
/dev/loop1         0      0         0  0 /home/zyga/snapd/potato (deleted) (deleted)   0     512
/dev/loop0         0      0         0  0 /home/zyga/snapd/potato (deleted)             0     512
```
The joke is that text formats are easy for humans and machines to parse.